pkg/build/build.go

```diff
@@ -99,3 +99,17 @@ func StoreCommonBinaries(kuberoot string, outroot string) {
 		}
 	}
 }
+
+// setReproducibilityEnv set the commit timestamp of kubeRoot for SOURCE_DATE_EPOCH env for reproducibility
+func setReproducibilityEnv(kubeRoot string) {
```

Comment:
Let's make the method name verbose so it's clear what it's doing, since it's a known concept https://reproducible-builds.org/specs/source-date-epoch/
maybe `setSourceDateEpoch`?